src/Symfony/Component/VarExporter/Tests/LazyGhostTraitTest.php

```diff
@@ -65,8 +65,10 @@ public function testUnsetPublic()
 
         $this->assertSame(["\0".TestClass::class."\0lazyObjectState"], array_keys((array) $instance));
         unset($instance->public);
-        $this->assertFalse(isset($instance->public));
         $this->assertSame(4, $instance->publicReadonly);
+        $this->expectException(\BadMethodCallException::class);
+        $this->expectExceptionMessage('__isset(public)');
+        isset($instance->public);
```

Comment:
This test case was asserting a behavior that relied on a work around with buggy side effects.
It's now asserting the correct behavior.
Fixing that incorrect behavior is how I spotted the root issue: other existing test cases that use the nullsafe operator where failing.
This means the work around for https://github.com/php/php-src/issues/12695 implemented here is already covered by tests.